java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -49,7 +49,7 @@ public static String getLinuxVersion() throws IOException {
             Process process = Runtime.getRuntime().exec("lsb_release -r");
             reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
             String line;
-            Pattern releasePattern = Pattern.compile("Release:\\s*(\\d+).*");
+            Pattern releasePattern = Pattern.compile("Release:\\s*(\\S+).*");
```

Comment:
The original regex parses OS release major version, but unfortunately `os-maven-plugin` only supports getting the full version.